MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -66,6 +67,7 @@ open class BaseInstructionsBannerView: UIControl {
     @objc func draggedInstructionsBanner(_ sender: Any) {
         if let gestureRecognizer = sender as? UIPanGestureRecognizer, gestureRecognizer.state == .ended {
             delegate?.didDragInstructionsBanner?(self)
+            stepListIndicatorView.isHidden = true
```

Comment:
If the top banner is tapped, the `stepListIndicatorView` is still visible. Both tapping and dragging should follow the same pattern.

Reply:
@bsudekum ✅